rust/worker/src/index/hnsw.rs

```diff
@@ -95,10 +95,12 @@ impl HnswIndexConfig {
             }
         }
 
-        let max_elements = get_metadata_value_as::<i32>(metadata, "hsnw:max_elements")?;
-        let m = get_metadata_value_as::<i32>(metadata, "hnsw:m")?;
-        let ef_construction = get_metadata_value_as::<i32>(metadata, "hnsw:ef_construction")?;
-        let ef_search = get_metadata_value_as::<i32>(metadata, "hnsw:ef_search")?;
+        let max_elements =
```

Comment:
Adds defaults. A future upgrade is to pull the defaults into a more sensible place.